### ISSUE TITLE
Issue with > character on IE

### ISSUE THREAD

**Issue by leonardo031195** [rubenv/angular-gettext#301]
Hi rubenv,

When I use a string that contains ">" on it, I get no translation on IE. Example:

`scope.sentence = gettextCatalog.getString('Sentence with > on it')`
`<span ng-bind="sentence"></span>`

On Chrome/Firefox, it works perfectly.

Thanks for your patience.

**Comment by SamanthaAdrichem** [comment]
I have the exact same issue with a different character and found out that IE converts the loaded JSON file. And changes & to &amp; so > will probably become &gt; The translation is then not found.

**Comment by SamanthaAdrichem** [comment]
That fixes it for me.

**Comment by danjarvis** [comment]
I was hitting this issue while using the `translate` filter:

`<span>{{'Save & Next' | translate}}</span>`

Since these were static strings I was able to switch to using the translate directive and it resolved the issue for me:

`<span translate>Save & Next</span>`

**Comment by danjarvis** [comment]
OK.

I just investigated this a bit further and the issue is not caused by IE / Edge converting the loaded JSON (that doesn't actually happen). The issue is that `setStrings()` is modifying each key if the `isHTMLModified` flag is set -- [see here.](https://github.com/rubenv/angular-gettext/blob/master/src/catalog.js#L172-L175)

This was added to account for IE uppercasing tag names and re-ordering HTML attributes.

**Comment by SamanthaAdrichem** [comment]
Yes, see my pull-request, which determines exactly the same, and notices that the fix isn't applied at a second location.

**Comment by bassoGeorge** [comment]
Hello,
I am also facing this issue and am in a situation where I can't use a workaround. Your fork works and fixes it  but would like this to be merged upstream. Can we prioritise pull request #312? Anything I can do to help ?

Thanks

**Comment by alexeykorsakov** [comment]
Hello

I am also interested pulling request #312. The suggested fix helped me.

Thanks

**Comment by korkmaz** [comment]
Hi,

#312 works for me too, It seems pull request is open for a while, Do you have a plan to merge it or Do you have another solution?

Thanks
